src/screens/Login/screens/Main.vue

```diff
@@ -173,12 +177,29 @@ export default {
         && containsNumber(this.password)
       );
     },
+
+    /**
+     * Valida de callbackPost é diferente de "/"
```

Comment:
erro de gramatica, deveria ser:
```suggestion
     * Valida se o callbackPost é diferente de "/"
```